src/app/layout/layout.component.ts

```diff
@@ -323,4 +359,14 @@ export class LayoutComponent implements OnInit, OnDestroy, AfterViewInit {
       }
     }
   }
+  updateTopBarPosition() {
+    const topBarElement = this.document.getElementById('campaign-top-bar');
```

Comment:
Can you please refactor the code using this.document.getElementBy  is a security risk if the ID provided is controlled by a malicious user, as it could lead to a Cross-Site Scripting (XSS) vulnerability.

Reply:
@skanderKha any updates for this PR ?

Reply:
@hichri-louay not yet @MohamedRayenJmili are you working on this issue ?

Reply:
i am testing the changes.